static/js/apps/eval_retrieval_generation/util.ts

```diff
@@ -66,3 +68,12 @@ export function processTableText(text: string): string {
   // created
   return text.replace(TABLE_DIVIDER_PATTERN, tableDivider);
 }
+
+// Get the first feedback stage to show for an eval type
+export function getFirstFeedbackStage(evalType: EvalType): FeedbackStage {
```

Comment:
curious will there be function like getSecondFeedbackStage later?

Reply:
Some ideas I'm playing with:
- getNextFeedbackStage(evalType, feedbackStage) & getPrevFeedbackStage(evalType, feedbackStage)
- turn feedbackNavigation into RigFeedbackNavigation and RagFeedbackNavigation and they each have the control over setting the feedback stage
- have a config that is a list of stages and as next or prev is clicked, we move through the list of stages

Reply:
I see. So sounds like this needs more work.